doc/HOWTO_RELEASE.rst.txt

```diff
@@ -1,4 +1,4 @@
-This file gives an overview of what is necessary to build binary releases for
+This file gives an overview of what is necessary to build binary releases forattr(cf.name()) = cf;
```

Comment:
Out of curiousity, what does ` forattr(cf.name()) = cf;` do?

Reply:
It proves one should carefully review commits before pushing a PR? Removing.